.github/workflows/check-docker-limit.yml

```diff
@@ -9,6 +9,12 @@ jobs:
     steps:
     - uses: actions/checkout@v4
 
+    - name: Install Yarn 4
+      run: |
+        corepack enable
+        corepack prepare yarn@4.6.0 --activate
+        yarn install
```

Comment:
I wonder if there is a way to modularize this to be used across all workflows? Then we would only have to update the version in one spot.